src/v2/Apps/Conversation/Components/ConversationCTA.tsx

```diff
@@ -39,6 +40,7 @@ export const ConversationCTA: React.FC<ConversationCTAProps> = ({
   conversation,
   openInquiryModal,
   openOrderModal,
+  createsOfferOrder,
```

Comment:
I see ConversationCTA doesn't need to receive this property, so if we remove it from line 24, then we can remove all the references to it.